.github/buildomat/jobs/tuf-artifact.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#:
+#: name = "helios / tuf-artifact"
+#: variety = "basic"
+#: target = "helios-latest"
+#: rust_toolchain = "nightly-2022-09-27"
+#: output_rules = [
+#:	"=/work/repo.zip",
+#: ]
+#:
+#: [dependencies.package]
+#: job = "helios / package"
+#:
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+cargo --version
+rustc --version
+
+COMMIT=$(git rev-parse HEAD)
+VERSION="0.0.0+git${COMMIT:0:11}"
```

Comment:
(not... necessarily a recommendation, but interested in your thoughts)

WDYT about using git tags to identify the version?

I *think* `git tag --contains HEAD` would return the tag, if one exists, and we could use that as the semver version?

I think your decision to use "0.0.0+git<commit>" is a *great* idea for development, but obviously, when we actually want to get around to publishing `0.0.1`, we need to store that semver value *somewhere*, for both individual components and for the system version.

Reply:
[`git describe`](https://git-scm.com/docs/git-describe) is probably the right play here then, because it gives either the tag of a commit, or a tag and the number of commits after that tag (and has some formatting play available).

Reply:
Also, while I have the bad idea train rolling through here: 

Should we embed some type of timestamp in here too, so that development versions of software can be lexicographically sorted, and we can automatically prefer "more-recent" ones?

Reply:
RFD some-number-i-can't-think-of declares artifact versions to be semver, I think. (I should probably enforce that in tufaceous.)

Reply:
Right, I meant as part of the `+...` semvar label